modules/nf-core/president/tests/main.nf.test

```diff
@@ -0,0 +1,101 @@
+import java.text.SimpleDateFormat
+
+nextflow_process {
+
+    name "Test Process PRESIDENT"
+    script "../main.nf"
+    process "PRESIDENT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "president"
+
+    test("president - nocompress") {
+
+        when {
+            process {
+                """             
+                input[0] = [[],file(params.test_data['sarscov2']['genome']['all_sites_fas'], checkIfExists: true)]
+                input[1] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                input[2] = ""
+                """
+
+            }
+        }
+
+        then {
+                assert process.success
```

Comment:
Is there any particular reason that the assertions are not in an assertAll block?

Reply:
there absolutely isn't, I did it now. :)